motionplan/motionPlanner.go

```diff
@@ -157,6 +157,10 @@ func Replan(ctx context.Context, request *PlanRequest, currentPlan Plan, replanC
 		return nil, err
 	}
 	// make sure there is no transformation between the PTG frame and World frame in the Solver frame
+	// TODO: RSDK-5391 this will trigger currently if we try to call Plan() on a robot which is currently executing another motion plan.
```

Comment:
I don't understand this comment. 

How is it that `len(sf.PTGSolvers()) > 0 && !spatialmath.PoseAlmostEqual(startPose, spatialmath.NewZeroPose())` will evaluate to `true` if a robot is calling `Plan()` concurrently?

Reply:
The contents of `request.StartConfiguration` are the result of calling `CurrentInputs()` on all `InputEnabled` components.

PTG bases are `InputEnabled`, but as the inputs describe relative rather than absolute positioning, if they are currently executing inputs from a `GoToInputs()` call, then they will report their progress through that arc, and 0s otherwise.

So if a PTG base is executing a Plan already, it will report a nonzero set of inputs, and thus a nonzero pose.

This is a feature, not a bug; we don't want to try to plan for things that are currently moving anyway.

Reply:
Thanks for explaining!